examples/EA/SOO/CMASimple.tpp

```diff
@@ -52,8 +52,8 @@ int main( int argc, char ** argv ) {
 	// Initialize the optimizer for the objective function instance.
 	//###begin<optimizer>
 	CMA cma;
-	cma.init( sphere );
-	cma.setSigma( 0.1 ); // Explicitely set initial globael step size.
+	cma.setInitialSigma(0.1);
+	cma.init( sphere, sphere.proposeStartingPoint()); // Explicitely set initial global step size.
```

Comment:
It is a detail, but the comment is copy&paste from two lines up, and this is not what's happending here.